pkg/scalers/cpu_memory_scaler.go

```diff
@@ -54,11 +55,14 @@ func parseResourceMetadata(config *ScalerConfig) (*cpuMemoryMetadata, error) {
 		averageValueQuantity := resource.MustParse(value)
 		meta.AverageValue = &averageValueQuantity
 	case v2beta2.UtilizationMetricType:
-		valueNum, err := strconv.ParseInt(value, 10, 32)
+		valueNum, err := kedautil.ParseNumeric(value, 32)
 		if err != nil {
 			return nil, err
 		}
-		utilizationNum := int32(valueNum)
+		utilizationNum, ok := valueNum.(int32)
+		if !ok {
+			return nil, fmt.Errorf("provided value for Utilization was not a valid integer")
```

Comment:
how about the below?

```suggestion
			return nil, fmt.Errorf("provided value for Utilization (%d) was not a valid 32-bit integer", valueNum)
```